django_socio_grpc/protobuf/json_format.py

```diff
@@ -36,3 +39,17 @@ def message_to_dict(message, **kwargs):
 def parse_dict(js_dict, message, **kwargs):
     kwargs.setdefault("ignore_unknown_fields", True)
     return ParseDict(js_dict, message, **kwargs)
+
+
+def _ConvertScalarFieldValue(value, *args, **kwargs):
+    """
+    We patch the json_format module to support UUIDs in ParseDict.
+    By default, having a UUID in a message will raise a TypeError.
```

Comment:
Add a todo that say we plan to use our own field and not DRF ones in the futur to deal with this serialisation before coming to this function